ts/torch_handler/base_handler.py

```diff
@@ -78,38 +81,101 @@ def initialize(self, context):
         self.manifest = context.manifest
 
         model_dir = properties.get("model_dir")
-        model_pt_path = None
+        self.model_pt_path = None
         if "serializedFile" in self.manifest["model"]:
             serialized_file = self.manifest["model"]["serializedFile"]
-            model_pt_path = os.path.join(model_dir, serialized_file)
+            self.model_pt_path = os.path.join(model_dir, serialized_file)
+
+        if self.model_pt_path.endswith("onnx"):
```

Comment:
Ah I see I like that idea is interesting because it makes it clear that the backends are completely different, this will make testing much easier on my end and checking end to end coverage much easier like for example does batching work? Does captum work with ONNX? etc..

I guess the tradeoff is because is we'll need another layer of handlers then we won't be able to reuse any code